plugins/github/test-fixtures/hosts.yml

```diff
@@ -0,0 +1,9 @@
+github.com:
+    user: testUser
+    oauth_token: github_pat_8BQUJmcVkoNo48YBxXjyF20gqFi7hYLdzhGmJUISVUCjyHOA0sdeO8Xmw7LqYY0Ng1wndPbww6fEXAMPLE
```

Comment:
I checked it. With any login method, it populates `oauth_token` property.

And as per our discussion, founds out that tokens with `github_pat_` prefix doesn't work with gh-cli.
```
error using api: non-200 OK status code: 401 Unauthorized body: "{\"message\":\"Personal access tokens with fine grained access do not support the GraphQL API\",\"documentation_url\":\"https://docs.github.com/graphql/guides/forming-calls-with-graphql#authenticating-with-graphql\"}"
```

So, we are going to check for `ghp_ ` prefix instead of `github_pat_`